packages/hardhat-core/src/internal/hardhat-network/stack-traces/vm-tracer.ts

```diff
@@ -106,11 +111,11 @@ export class VMTracer {
 
         trace = createTrace;
       } else {
-        const toAsBn = new BN(message.to.toBuffer());
+        const toAsNumber = bufferToBigInt(message.to.toBuffer());
```

Comment:
`toAsBigInt`?

Reply:
Renamed in https://github.com/NomicFoundation/hardhat/pull/3082/commits/a44ca54f18b180e87d3bfe62055eb179f09aa1d0